lib/universal-search.js

```diff
@@ -32,6 +34,7 @@ Search.prototype = {
     win.universalSearch = win.universalSearch || {}
 
     this._hideSearchBar(win);
+    this._enableSearchSuggestions(win);
```

Comment:
Opting for the simple approach here: set it to true on enable/install, reset on disable/uninstall. We can revisit after launch if it becomes a problem.